flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -12,7 +12,7 @@ import AppKit
 // swiftlint:disable:next type_body_length
 public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
-    private static let nativeClientName = "sentry.cocoa.flutter"
+    private static let nativeClientName = "sentry.native.cocoa.flutter"
```

Comment:
This change isn't needed, the issue is only about the Sentry Native for Android NDK

Reply:
So we don't need to align on the coco side that the naming is the same?

Reply:
Sentry the confusion, when I meant Sentry Native isn't about Android and iOS, its about https://github.com/getsentry/sentry-native
SentryNative for NDK support.- Android.